board-viewer.html

```diff
@@ -442,7 +442,10 @@
   ],
 
   keyBindings: {
-    'esc': '_onEscPressed'
+    "esc": "_onEscPressed",
+    "f1:keydown": "_onF1Entering",
+    "f1:keyup": "_onF1Leaving",
```

Comment:
@jketreno I can add the listener over 'Ctrl' key-event, but the problem that I'm noticing is that holding 'Ctrl' and  making a 'click' the navigator stops the addition of components, because the client browser opens its own secondary menu, like the shown below. 

Chrome (Ctrl+click):

![screen shot 2018-01-19 at 14 23 22](https://user-images.githubusercontent.com/8356225/35170116-ad2686ea-fd24-11e7-968e-8daa01152cb9.png)

FireFox (Ctrl+click):

<img width="259" alt="screen shot 2018-01-19 at 14 23 43" src="https://user-images.githubusercontent.com/8356225/35170143-c5dc7f5a-fd24-11e7-8407-b2a291bcfc68.png">

Safari (Ctrl+Click):

![screen shot 2018-01-19 at 14 42 17](https://user-images.githubusercontent.com/8356225/35170814-116d26c0-fd27-11e7-984f-351fc422765e.png)

Right Now: I can not add multiple components since my desktop, listening Ctrl, because when I make the click, the menu is opened, regardless the browser. Preserving the Ctrl listening we are reducing this functionality only to touch devices (I'm not really sure if this happens making a tap [touch device]).

 But, If were posible to stop original client navigator functionality of ctrl+click, is it ok? is it worthy to keep Ctrl+click listener in the board-viewer?

Nevertheless, F1 key It seems to me accurate since the WAI-ARIA-PRACTICES perspective (https://www.w3.org/TR/wai-aria-practices/#kbd_general_binding).

Other proposal can be the application of the pseudo patch, but using 'meta' key (Windows for windows keyboards, Cmd for MAC keyboards), Instead of ctrl. It seems that Meta+click does not launches any client browser or OS functionality.

I think I have 3 options identified to make this change:

1. Apply pseudo patch with 'Ctrl' + click. (Not agree due the combination of key and click is a client browser reserved functionality and is not within WAI-ARIA practices).
2. Apply pseudo patch with 'Meta' + click. (Semi-agree, this is only a NOT best practice...).
3. Letting F[0-9] key (Agree, do not interfieres with other OS/browser functionality and is within a w3c set of best practices).

But at the end I will do the one you choose, so only to confirm that, I my preferred one is the option '3', what is yours ? the option '1'?

Reply:
Are you on a system that is emulating right mouse clicks via ctrl-click?

On Windows and Linux systems, Ctrl+click is the standard multi-select keyboard shortcut.

On Mac, it's Command+click.

To support the Mac, in the keydown and keyup, you can check event.keyCode to set the 'ctrlPressed' proprety... something like:

```javascript
var key = event.keyCode.
if (key == 91 || key == 93 || key == 224) {
  this.ctrlKey = (event.type == 'keydown');
  return;
}
this.ctrlKey = event.ctrlKey;
```
Users on that Command systems (Mac) would then use the expected keyboard shortcut for multi select (Command+click) and on Windows and Linux, they can continue using the default system shortcut there (Control + click)

By following the standards, we don't have to document the behavior; it will do what users expect based on how the rest of their system works. If a user is on Windows and they've remapped Ctrl-click to emulate the right mouse button, there is no standard and they won't be able to mulit-select anywhere in their operating system using the keyboard.

Reply:
@jketreno I got it, I was missing that part of the picture... I think now it makes sense, I will send the changes for support cmd/ctrl.